tests/unit/sagemaker/jumpstart/test_notebook_utils.py

```diff
@@ -63,7 +64,7 @@ def test_list_jumpstart_scripts(
     patched_generate_jumpstart_models.assert_called_once_with(
         **kwargs, sagemaker_session=DEFAULT_JUMPSTART_SAGEMAKER_SESSION
     )
-    patched_get_manifest.assert_called_once()
+    assert patched_get_manifest.call_count == 3
```

Comment:
curious: why 3 not 2?

Reply:
This should be 2, updated in the latest commit. Previously it was 3 because of the implementation of this [thread](https://github.com/aws/sagemaker-python-sdk/pull/4467#discussion_r1509071252), but since I go with option2 this should be 2.